### DISCUSSION TITLE
188

### DISCUSSION THREAD

**Discussion by giscus** [tw93/Weekly#160] [General]
# 潮流周刊第188期 - 水墨蓝天

封面图拍摄于周五早下班后拍到的蓝天图，很有水墨画的感觉，很美，顿时心情很好，也许这就是知足的感觉。

https://weekly.tw93.fun/posts/188-%E6%B0%B4%E5%A2%A8%E8%93%9D%E5%A4%A9

**Comment by xcdmrCHP** [comment]
> 随便写写：什么是好的设计

不太理解最后一句：`可以被复制的、可以成批出现的`
好的设计不应该是独特的、脱颖而出的吗？

**Reply by shellGrace** [reply]
在技术的角度，可以理解为 可复用性 吧，可复用性高可以提高生产效率

**Reply by xcdmrCHP** [reply]
或者是说，指定是通用性，可移植性？
比如使用规范的figma或者psd格式；可在广告印刷，前端页面上显示

**Reply by tw93** [reply]
可以被复制:
好的设计通常有清晰的原则和逻辑,可以被其他人理解和学习。
它不是一次性的灵感迸发,而是有方法可循的。
可复制性使得好的设计可以被广泛应用,而不仅限于少数人。

可以成批出现:
优秀的设计可以被规模化生产,满足大众需求。
它不是稀有的艺术品,而是可以惠及更多人的实用设计。
批量生产也意味着设计本身具有一定的普适性。

被大众称赞:
好的设计应该能够得到普通用户的认可,而不仅仅是设计师圈子的赞誉。
它需要在美学和功能性之间找到平衡,既吸引眼球又实用。
大众的认可也反映了设计与当前文化、审美趋势的契合度。

**Reply by jeasonnow** [reply]
真正的艺术，不在于取悦少数，而在于其普世之美。

简单点说，能让所有人都能喜欢/使用/复制的设计，才称得上优秀的设计。

**Reply by cMing1997** [reply]
我个人认为，好的设计是 一套作品是成体系的，可以有自己的独特性。但是相似的东西设计出来得有逻辑性。另一个就是好的设计一定是方便使用，符合大多数人的使用以及阅读习惯的

**Comment by 0522skylar** [comment]
发现93很喜欢拍天空！

**Reply by tw93** [reply]
的确 可能是刚好天气好 心情好 还有一种原因是 天空好看的时候色彩渲染很适合封面图

**Comment by ClishWang** [comment]
页面有报错，点击切换黑暗模式无效

**Reply by tw93** [reply]
谢谢反馈已经修复了

**Comment by tonytian2000** [comment]
楼主还是投资小白啊，欢迎来www.zero2me.com学习下投资理论

**Comment by josangmi** [comment]
很多之前玩股票的都不玩了，真的不适合大部分人，起码现在我们能买到的这个。

**Reply by tw93** [reply]
美股美股美股
